api/v1alpha1/freight_helpers_test.go

```diff
@@ -26,10 +26,9 @@ func freightByCurrentStagesIndexer(obj client.Object) []string {
 		return nil
 	}
 	currentStages := make([]string, len(freight.Status.CurrentlyIn))
-	var i int
+	currentStages := make([]string, 0, len(freight.Status.CurrentlyIn))
```

Comment:
Something went wrong here when the suggestion was auto-applied.

Reply:
Will fix.